ciao-cli/main.go

```diff
@@ -1031,7 +1038,12 @@ func main() {
 		if *imageName == "" {
 			fatalf("Missing required -image-name parameter")
 		}
-		createTenantImage(*identityUser, *identityPassword, id, *imageName)
+		if *imageVisibility != "private" && *imageVisibility != "public" {
+			fatalf("image-visibility can be either 'private' or 'public'")
+		}
+		createTenantImage(*identityUser, *identityPassword, id, *imageName,
+			*imagePath, *containerFormat, *diskFormat, *imageVisibility, *imageTags,
+			*minDiskGigabytes, *minRAMMegabytes, *protected)
```

Comment:
There is also a type CreateOpts struct in "github.com/rackspace/gophercloud/openstack/imageservice/v2/images"

so same question as above, to avoid code duplication should I pass a CreateOpts struct to createTenantImage ?